fulp_modules/features/antagonists/bloodsuckers/code/structures/bloodsucker_crypt.dm

```diff
@@ -605,7 +619,9 @@
 /obj/structure/bloodsucker/bloodthrone/post_buckle_mob(mob/living/target)
 	. = ..()
 	target.pixel_y += 2
-	update_appearance(UPDATE_OVERLAYS)
+	if(dir == NORTH)
+		layer = ABOVE_MOB_LAYER
```

Comment:
What does this do? Also it doesn't seem to revert itself if it gets rotated?

Reply:
I did this in order to correct an odd visual bug I noticed where mobs buckled into the chair would be rendered over it if the chair was facing north. Also I think it does revert on rotation:

![image](https://github.com/user-attachments/assets/d46636b2-3482-4e92-93f6-bda4f528de00)

Actually (now that I'm looking at this image) I see the problem. The layer only changes on rotation if the chair has a buckled mob...

...I'll implement a simpler fix by just removing the second `has_buckled_mobs()` call.